### ISSUE TITLE
[Class hierarchy]

### ISSUE THREAD

**Issue by pgaudet** [obophenotype/cell-ontology#818]
**CL term**
gamete (CL:0000300)

**Suggested revision and reasons**

Please remove 'capable of' some 'multicellular organismal reproductive process' 

This is creating problems in GO, see https://github.com/geneontology/go-ontology/issues/16329

@ValWood

Thanks, Pascale

**Comment by nicolevasilevsky** [comment]
@pgaudet this is inherited from germ cell, which has the subclassOf axiom 'capable of' some 'multicellular organismal reproductive process'. If we remove it from there, that addresses your issue, but I am not sure if this is the correct approach?

**Comment by dosumis** [comment]
surely 'capable of part of' ? 

@pgaudet - would that fix the GO problem?

**Comment by balhoff** [comment]
@dosumis see my comment here: https://github.com/geneontology/go-ontology/issues/20362#issuecomment-726391379

For @ValWood's  purposes there can't be any existential link to a term excluded from fission yeast. I think we should consider whether `'multicellular organismal process' never_in_taxon 'Schizosaccharomyces pombe'` is really valid.

**Comment by dosumis** [comment]
Surely this is a GO problem?   I don't like the idea that perfectly valid axioms need to be deleted from CL because they conflict with GO build issues. This could bite us in other cases where we make links to GO.  If I understand the issue correctly, it could also cause problems in other situations where we use the same strategy for creating taxon subsets.

**Comment by balhoff** [comment]
> Surely this is a GO problem?

I agree, probably so, or at least we need to decide which is wrong:
- `'germ cell' SubclassOf 'capable of' some 'multicellular organismal reproductive process'` (CL)
- `'multicellular organismal process' never_in_taxon 'Schizosaccharomyces pombe'` (GO)

**Comment by pgaudet** [comment]
I think `multicellular organismal process' never_in_taxon 'Schizosaccharomyces pombe`  is right

`'germ cell' SubclassOf 'capable of' some 'multicellular organismal reproductive process'`
-> change to capable of' some 'reproductive process'`? 

Thanks, Pascale

**Comment by nicolevasilevsky** [comment]
It looks like germ cell already has the SubClass Of (Anonymous Ancestor) 'capable of' some 'reproductive process', and therefore gamete does too. So maybe removing the SubClass Of axiom from 'germ cell' is maybe okay, if what we want is for 'germ cell' to have the axiom 'capable of' some 'reproductive process'?

**Comment by pgaudet** [comment]
As far as GO is concerned, we would like to see the multicellular organism aspect removed. Reproduction is fine.

**Comment by dosumis** [comment]
I don't see any harm in doing this.  But I'd really like a better understanding of the problem.  We should be able to state this as a simple set of biologically meaningful statements that together => a contradiction.  What minimal set of assertions do we need to add to these to tell that complete story:

- `'germ cell' SubclassOf 'capable of' some 'multicellular organismal reproductive process'` (CL)
- `'multicellular organismal process' never_in_taxon 'Schizosaccharomyces pombe'` (GO)

?

**Comment by ValWood** [comment]
so 
'germ cell' SubclassOf 'capable of' some 'multicellular organismal reproductive process'
is not always true.
pombe have germ cells but are never multicellular?

**Comment by dosumis** [comment]
> pombe have germ cells 

Not questioning the biology here, but interested if it is directly asserted or inferred anywhere?  If so, can you say how?

**Comment by pgaudet** [comment]
This was in the original comment geneontology/go-ontology#16329

This is what is creating problems for GO

**Comment by dosumis** [comment]
I've tried working through the 73 comments on that ticket, looking at the various terms in QuickGO and trying to puzzle how this inference happens, but I'm still confused.  If this is coming from the slightly hack-y way that taxon subsets are constructed (see comment that Jim linked to), I'm worried that this indicates a wider problem we should be aware of.

I don't want to hold up your edits though, if there's a solution to this particular problem that we're all happy with.  `reproductive process` strikes me as very vague.  I'm wondering whether we should be linking germ cell to 'fertilization` or to something that groups `fertilization` and `conjugation with cellular fusion`

**Comment by dosumis** [comment]
Yep. Seen that.  It's what worries me.   Maybe I should just have a chat with Jim offline so I can understand properly. For the fix, would you be happy with us using a more specific term that does not come with an inference of multicellularity?

**Comment by ValWood** [comment]
@dosumis  that was the resolution . I'm not using the polyspermy term. A new term was created with no multicellular parentage.

**Comment by ValWood** [comment]
we now have
GO:0140538 +name: negative regulation of conjugation with zygote
https://www.ebi.ac.uk/QuickGO/term/GO:0140538
which I should hopefully be able to access soon

**Comment by pgaudet** [comment]
@ValWood were you saying that you dont need the edit to CL ? (here https://github.com/geneontology/go-ontology/issues/16329#issuecomment-737180120 ) 

sorry @dosumis looks like there is some miscommunication !

**Comment by dosumis** [comment]
I had chat with Jim to get my head around the issues.

Plan
Move axiom on germ cell -> capable_of_part_of fertilization.

we noticed an apparent discrepancy between the GO BP main branch and the regulation branch

Fertlization and conjugation with cellular fusion are sibling terms under sexual reproduction

![image](https://user-images.githubusercontent.com/112839/101057362-de35fc80-3583-11eb-8a24-62b87e5da6e8.png)

But in the regulation branch we have 

![image](https://user-images.githubusercontent.com/112839/101057655-2f45f080-3584-11eb-979b-69b03bc4180a.png)

Should fertilization be the more general term?  Or should GO have a new general term that is a superclass of conjugation with fusion?  If that was added, I'd be happy to use it in CL in place of fertilization.
